Sources/Plasma/Apps/plClient/external/render_svg.py

```diff
@@ -247,7 +248,12 @@ def render_voice_icons(inpath, outpath):
 	outpath = os.path.expanduser(options.outpath)
 	inpath = os.path.expanduser(options.inpath)
 
-	if not os.path.exists(outpath):
+	# Ensure that the render directory is empty, preventing old renders from junking the output
+	if os.path.exists(outpath):
+		print("Cleaning render directory...")
+		for i in glob(os.path.join(outpath, "*.png")):
```

Comment:
I forget the actual file structure - Is there any reason to preserve any file in the outpath? Can we just purge it entirely? That way if the renders are changed to a different format, nothing breaks here.

Reply:
I initially considered deleting the entire directory, but I decided that since the script is invocable with an arbitrary output path outside of the build process, simply deleting the entire directory would a nasty GOTCHA.

Reply:
Fair enough. On that case it'd be good to DRY up that "*.png" with the rendering code. But that's outside the scope of this PR